docs/SECURITY.md

```
@@ -0,0 +1,267 @@
+**Last Updated:** 2025-12-06
+
+---
+
+## Recent Security Fixes
+
+### 2025-12-06: Pillow CVE-2024-28219
+
+**Vulnerability:** CVE-2024-28219 in Pillow
+**Severity:** [To be verified - check CVE database]
+**Affected Versions:** Pillow < 10.3.0
+**Fixed Version:** Pillow >= 10.3.0
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)